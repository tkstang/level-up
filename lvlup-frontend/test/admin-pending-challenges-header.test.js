import Header from '../src/components/Admin/admin-main-view/pending-challenges/header';
import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import thunk from 'redux-thunk';
import configureMockStore from 'redux-mock-store';
import lvlupApp from '../src/reducers/index';

const middlewares = [thunk];
const mockStore = configureMockStore(middlewares);

test('Header should render', () => {
  const store = mockStore({ lvlupApp });
  const approveChallenge = shallow(
    <Header store={store} />,
  );
  expect(shallowToJson(approveChallenge)).toMatchSnapshot();
});
