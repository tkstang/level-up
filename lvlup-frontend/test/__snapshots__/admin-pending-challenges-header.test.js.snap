exports[`test Header should render 1`] = `
<h2
  className="header">
  Pending Challenge Submissions
</h2>
`;
